whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -119,8 +120,12 @@ public RpslObject apply(final RpslObject rpslObject) {
     }
 
     private boolean isOverrideAuthenticated(final ObjectType objectType){
-        return overrideCredentialValidator != null && overrideCredentialValidator.isAllowedAndValid(isTrusted,
-                userSession, overrideCredential, objectType);
+        try {
+            return overrideCredentialValidator != null && overrideCredentialValidator.isAllowedAndValid(isTrusted,
+                    userSession, overrideCredential, objectType);
+        } catch (EmptyResultDataAccessException e){
```

Comment:
Sorry I should have pointed put wth earlier. comment but can you  all exceptions? There can be incorrectrSizeesultExce[tion too

Reply:
Updated